docs/workflows/genomic_characterization/theiaprok.md

```diff
@@ -372,6 +372,9 @@ All input reads are processed through "[core tasks](#core-tasks-performed-for-al
 | merlin_magic | **shigatyper_docker_image** | String | The Docker container to use for the task | us-docker.pkg.dev/general-theiagen/staphb/shigatyper:2.0.5 | Optional | FASTA, ONT, PE, SE |
 | merlin_magic | **shigeifinder_docker_image** | String | The Docker container to use for the task | us-docker.pkg.dev/general-theiagen/staphb/shigeifinder:1.3.5 | Optional | FASTA, ONT, PE, SE |
 | merlin_magic | **sistr_docker_image** | String | The Docker container to use for the task | us-docker.pkg.dev/general-theiagen/biocontainers/sistr_cmd:1.1.1--pyh864c0ab_2 | Optional | FASTA, ONT, PE, SE |
+| merlin_magic | **sistr_cpu** | Int | The number of CPU cores to allocate for the task. | 8 | Optional | FASTA, ONT, PE, SE  |
```

Comment:
super nit-picky: can you order these outputs in alphabetical order with the surrounding? 

our ordering priority is 1) 1st column, then 2) 2nd column